Kudu.FunctionalTests/GitDeploymentTests.cs

```diff
@@ -496,8 +497,8 @@ public void NodeWithSolutionsUnderNodeModulesShouldBeDeployed()
                     Assert.Equal(DeployStatus.Success, results[0].Status);
 
                     var log = GetLog(appManager, results[0].Id);
-                    Assert.Contains("18.3.0", log);
-                    Assert.Contains("8.11.0", log);
+                    Assert.Contains("18.3.0", log); // Node js version.
```

Comment:
two things for kudu coding style - 1) comment always on the different line  2) please squash to a single commit.